app/javascript/src/components/Dashboard/Notes/Card.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Avatar, Dropdown, Tooltip, Tag, Typography } from "neetoui";
+
+import {
+  calculateCreatedAgo,
+  formatDateRelativeToNow,
+} from "components/Dashboard/utils";
+
+import { USER_NAME, USER_AVATAR_URL } from "./constants";
+
+const Card = ({ note }) => {
+  const { Menu, MenuItem } = Dropdown;
```

Comment:
Let's keep this outside of the component so that we avoid running this statement when the component re-renders.